react/components/OneTapSignin.js

```diff
@@ -98,7 +100,7 @@ const OneTapSignin = ({
         }}
       />
       <form className="dn" ref={formRef}>
-        <input name="account" value={account} />
+        <input name="account" value={account} onChange={() => {}} />
```

Comment:
As the error suggested, I guess readOnly is more appropriate, given that the component is not controlled

![image](https://user-images.githubusercontent.com/284515/83805556-76f7c800-a686-11ea-85b1-a799ec4a717d.png)

```suggestion
        <input name="account" value={account} readOnly />
```

Reply:
True, my bad